lambdas/handlers/bulk_upload_handler.py

```diff
@@ -23,28 +18,7 @@ def lambda_handler(event, _context):
         try:
             logger.info(f"Processing message {index} of {len(event['Records'])}")
             bulk_upload_service.handle_sqs_message(message)
-        except (InvalidMessageException, LGInvalidFilesException) as error:
-            handle_invalid_message(invalid_message=message, error=error)
-
-
-def handle_invalid_message(invalid_message: dict, error=None):
-    # Currently we just send the invalid message to invalid queue.
-    # In future ticket, will change this to record errors in dynamo db
-    invalid_queue_url = os.environ["INVALID_SQS_QUEUE_URL"]
-    sqs_service = SQSService()
-
-    new_message = {"original_message": invalid_message["body"]}
-    if error:
-        new_message["error"] = str(error)
-
-    try:
-        nhs_number = invalid_message["messageAttributes"]["NhsNumber"]["stringValue"]
-    except KeyError:
-        nhs_number = ""
-
-    sqs_service.send_message_with_nhs_number_attr(
-        queue_url=invalid_queue_url,
-        message_body=json.dumps(new_message),
-        nhs_number=nhs_number,
-    )
-    logger.info(f"Sent message to invalid queue: {invalid_message}")
+        except Exception as error:
```

Comment:
Should we be catching generic errors here? Think it's best practice to only catch what we know it could throw

Reply:
Thanks!  Agree that it is bad to catch the generic error type.
I have changed this part to catch a list of several possible errors.